libraries/ui-ios/AppBar/index.jsx

```diff
@@ -59,7 +59,7 @@ const AppBar = ({
     };
   }, [contentRef, observer]);
 
-  const sectionClasses = classnames(styles.outer, classes.outer);
+  const sectionClasses = classnames(styles.outer, classes.outer, 'ui-ios__app-bar');
```

Comment:
what is this className for?

Reply:
a bunch of the PWA 6 components have those "stable" human readable classes. this class was migrated from here
https://github.com/shopgate/pwa/blob/releases/v6.23.0/libraries/ui-ios/AppBar/index.jsx#L71
it think they are used by extensions to modify styling. migration of those classes was one of the major tasks of the migration process.